Core/Graphics/Renderer.cpp

```diff
@@ -1,6 +1,6 @@
 #include "Renderer.h"
 #include "Core/Graphics/Types.h"
-#include <SDL2/SDL.h>
+#include <SDL2/SDL_video.h>
```

Comment:
Added this as we are using only SDL_GetWindowSize thorughout the file which is under `SDL_video.h` header